plugins/modules/ec2_instance.py

```diff
@@ -2271,6 +2282,7 @@ def main():
                 instance_metadata_tags=dict(choices=["disabled", "enabled"], default="disabled"),
             ),
         ),
+        additional_info=dict(type='str'),
```

Comment:
The type listed in the arguments doesn't match the type listed in the return and parameters documentation.

https://github.com/ansible-collections/amazon.aws/actions/runs/6798846893/job/18485232245?pr=1828

```
ERROR: plugins/modules/ec2_instance.py:0:0: doc-type-does-not-match-spec: Argument 'additional_info' in argument_spec defines type as 'str' but documentation defines type as 'dict'
```

Reply:
fixed now.